packages/x-charts/src/ChartsTooltip/DefaultChartsAxisTooltipContent.tsx

```diff
@@ -18,7 +18,21 @@ function DefaultChartsAxisTooltipContent(props: ChartsAxisContentProps) {
   if (dataIndex == null) {
     return null;
   }
-  const axisFormatter = axis.valueFormatter ?? ((v) => v.toLocaleString());
+
+  const utcFormatter = (v: any): string => {
```

Comment:
To avoid the `any` you can modify [here](https://github.com/mui/mui-x/blob/next/packages/x-charts/src/ChartsTooltip/ChartsAxisTooltipContent.tsx/#L34)
to set `string | number | Date | null`